### PR TITLE
Better explanations and more examples of how the Reactor calls functions

### PR DESCRIPTION
**Pull request by whiteinge**
Refs #20841

### CODE REVIEW

File: doc/topics/reactor/index.rst [hunk 1]
```diff
@@ -173,52 +173,46 @@ rendered SLS file (or any errors generated while rendering the SLS file).
...
-* :py:class:`~salt.wheel.WheelClient` calls Wheel modules locally on the
-  master.
-
-The :strong:`state declaration` field takes a reference to the function to call
-in each interface. So to trigger a salt-run call the :strong:`state
-declaration` field will start with :strong:`runner`, followed by the runner
-function to call. This means that a call to what would be on the command line
-:strong:`salt-run manage.up` will be :strong:`runner.manage.up`. An example of
-this in a reactor formula would look like this:
+The Reactor runs on the master and calls functions that exist on the master. In
```
Comment (cachedout) [inline doc/topics/reactor/index.rst]: Reactors can run on minions, as of somewhat recently.

Comment (whiteinge) [inline doc/topics/reactor/index.rst]: That will complicate this explanation. BACK TO THE DRAWING BOARD!

**Comment by whiteinge** [comment]
On a more serious note, let's get these changes in here and we can update this doc on whatever branch the minion-side reactor stuffs has gone into.

**Comment by cachedout** [comment]
Roger that. Merging!

**Pull request merged**